docs/conf.py

```diff
@@ -260,7 +260,12 @@ def rstjinja(app, docname, source):
         source[0] = rendered
 
 
+# JSOC email os env
+os.environ["JSOC_EMAIL"] = "nabil.freij@gmail.com"
```

Comment:
The address we are using in tests at the moment is `jsoc@cadair.com`, it's a blackhole.

Reply:
Fixed.